src/ui/Container/index.tsx

```diff
@@ -15,6 +15,6 @@ export const Container = ({
     style={[styles.container, { paddingBottom: getNavbarHeight() + 16 }, style]}
     {...props}
   >
-    {children}
+    <SafeAreaView>{children}</SafeAreaView>
```

Comment:
nit: What do you think of creating a CozyView that wraps SafeArea view on iOS and nothing on android? Can it makes sense?

Reply:
I just did the test about double margins. The SafeAreaView is able to adapt to container's position

Here is a version with the container's height=100% (green = container, red = SafeArea)
<img width="358" alt="image" src="https://user-images.githubusercontent.com/1884255/199740751-30c364a6-7c88-4bbf-965f-679bf146fd25.png">

Here is a version with the container's height=99%
<img width="351" alt="image" src="https://user-images.githubusercontent.com/1884255/199740827-eab1fed3-4591-43a6-be50-d97286ec1f8b.png">

Here is a version with the container padding bottom being less than the Safe area margin
<img width="357" alt="image" src="https://user-images.githubusercontent.com/1884255/199741130-12620afc-ed17-4764-86c5-de56fcb6bf1f.png">

The `déverrouiller` button won't move if the container's padding is moving but stays under the SafeArea limit.

The behavior is the same for the top margin.

So it seems like we can safely keep the SafeAreaView in all our view without risk of margin conflict.